tutorials/clinical/10_ieeg_localize.py

```diff
@@ -207,7 +207,8 @@ def plot_overlay(image, compare, title, thresh=None):
 #         - Load the two scans from the command line using
 #           ``freeview $MISC_PATH/seeg/sample_seeg/mri/T1.mgz
 #           $MISC_PATH/seeg/sample_seeg_CT.mgz``
-#         - Navigate to the upper toolbar, go to ``Tools>>Transform Volume...``
+#         - Navigate to the upper toolbar, go to
+#           :menuselection:`Tools --> Transform Volume...`
```

Comment:
why only this one?  There is a `Tools -> transform volume` above as well as `Freeview -> Preferences -> General -> Cursor style -> Long` (again, both in the "aligning to ACPC" section)